policyengine_canada/variables/gov/cra/benefits/old_age_security_pension/old_age_security_pension.py

```diff
@@ -1,14 +1,19 @@
 from policyengine_canada.model_api import *
 
-# SPSD/M 29.0: imioas or imoasmax, I'm not sure. I think imioas is post-clawback?
-
-
+# SPSD/M 29.0: imioas or imoasmax, I'm not sure. I think imioas is post-repayment?
 class old_age_security_pension(Variable):
     value_type = float
     entity = Person
-    label = "Old age security pension pre-clawback"
-    documentation = "The OAS amount a person is eligible for prior to the clawback tax. See SPSD/M 'imoasmax'."
+    label = "Old age security pension pre-repayment"
+    documentation = "The OAS amount a person is eligible for prior to the repayment tax. See SPSD/M 'imoasmax'."
     unit = CAD
     definition_period = YEAR
-    adds = ["old_age_security_pension_pre_clawback"]
```

Comment:
Let's actually keep the `adds`/`subtracts` part so that users can drill into each part in the app. If both elements are rounded the result should be too, and we round when displaying it. We might want to add rounding as a variable attribute though @nikhilwoodruff

Reply:
Oddly when I do it the `adds`/`subtracts` way it doesn't return the final pension amount to two decimals, even though the two components are both rounded. 

So this passes:

```
    old_age_security_pension_pre_repayment: 9048.60
    old_age_security_pension_repayment: 2735.85
  #  old_age_security_pension: 6312.75

```

But this fails with messages `old_age_security_pension@2022: [6312.7495] differs from 6312.75 with an absolute margin [0.00048828] > 0`

```
    old_age_security_pension_pre_repayment: 9048.60
    old_age_security_pension_repayment: 2735.85
    old_age_security_pension: 6312.75

```

Reply:
Strange, that looks like too big a delta to be a floating point issue. Could you add `absolute_error_margin: 0.01` to the yaml test ([example](https://openfisca.org/doc/coding-the-legislation/writing_yaml_tests.html#example)) and file an issue?